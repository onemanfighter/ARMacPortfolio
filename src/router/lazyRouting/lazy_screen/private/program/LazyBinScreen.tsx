import { lazy } from 'react';
import { LazyProvider } from '../../../component';

const Bin = lazy(
  () => import('../../../../../screens/private/program/Bin/Bin'),
);

const LazyBinComponent = () => {
  return (
    <LazyProvider>
      <Bin />
    </LazyProvider>
  );
};

export { LazyBinComponent };
