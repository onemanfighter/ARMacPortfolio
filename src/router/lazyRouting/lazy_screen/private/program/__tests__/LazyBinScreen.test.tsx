import { render, screen, waitFor } from '@testing-library/react';
import { LazyBinComponent } from '../LazyBinScreen';

describe('LazyBinComponent', () => {
  it('should render correctly to match snapshot', async () => {
    const { container } = render(<LazyBinComponent />);

    await waitFor(() => expect(screen.getByText('Bin')).toBeDefined());
    expect(container).toMatchSnapshot();
  });
});
