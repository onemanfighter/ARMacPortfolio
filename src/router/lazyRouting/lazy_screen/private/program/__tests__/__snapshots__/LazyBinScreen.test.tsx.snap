// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LazyBinComponent should render correctly to match snapshot 1`] = `
<div>
  <div>
    <h1>
      Bin
    </h1>
  </div>
</div>
`;
